src/pages/docs_sdk_forgot-password.html

```diff
@@ -0,0 +1,44 @@
+<div class="section">
+  <h2><a href="/docs/">Docs</a> : <a href="/docs/sdk/">SDK</a> : forgotPassword</h2>
+
+  <p><span class="font-semibold">forgotPassword</span> lets you send a temporary password to the user's email address. This API will return a promise that gets resolved once the email with the temporary password has been sent.</p>
+
+  <pre>
+    <code class="language-javascript">userbase.forgotPassword({ username })
+      .then((user) => {
+        // email with temporary password sent
+      })
+      .catch((e) => console.error(e))</code>
+  </pre>
+
+  <h3 id="params">Parameters</h3>
+
+  <ul>
+    <li>
+      <span class="field">username</span> [string | Len: 1-50] - The username for the account to send a temporary password to.
+    </li>
+  </ul>
+
+  <h3 id="params">Notes</h3>
+
+  <ul>
+    <li>
+      Resetting a user's password is possible, but only when the user has provided an email address during <a href="/docs/sdk/sign-up/">signUp</a> or <a href="/docs/sdk/update-user/">updateUser</a>, and when the user has previously signed in with the <span class="field">rememberMe</span> option set to 'local'. In this case, the user will have the encryption key saved in the browser's local storage, and the user will be able to regain access to the account by getting a temporary password via email. The user must still have access to a previously used device in order to be able to reset the password. Therefore, if you want to allow your users to reset their password, make sure to set <span class="field">rememberMe</span> to 'local' during <a href="/docs/sdk/sign-up/">signUp</a> and <a href="/docs/sdk/sign-in/">signIn</a>, and make sure you require an email address during <a href="/docs/sdk/sign-up/">signUp</a> and <a href="/docs/sdk/update-user/">updateUser</a>. We still recommend that you inform your users that since their data is end-to-end encrypted, they should take care to store their password in a safe place, such as a password manager. Recovery will not be possible if the user loses access to all the previously used devices.
```

Comment:
Maybe we want to add a similar note to rememberMe in signUp and signIn? Some might not see this note here in forgotPassword before making their decision on rememberMe